docs/application/web/get-started/wearable-widget/first-app-widget.md

```diff
@@ -422,8 +416,7 @@ To run the application on the target device:
 
         Like with the [emulator](#viewer), you can add the widget to the home screen to easily access it.
 
-    > **Note**
-    >
+    > [!NOTE]
     > The application is launched using the default debug run configuration. To create and use another configuration:
     > 1.  In the `Project Explorer` view, right-click the project and select `Run As > Run Configurations`.
```

Comment:
Just a question.

Is there any reason that these sentences in note looks in green?

![image](https://user-images.githubusercontent.com/9958374/71966316-affd4f00-3244-11ea-9a95-a021f345e768.png)